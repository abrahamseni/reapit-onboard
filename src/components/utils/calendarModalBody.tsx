import { format, getDay, parse, startOfWeek } from 'date-fns'
import enAU from 'date-fns/locale/en-AU'
import * as React from 'react'
import { Calendar, dateFnsLocalizer, Event, SlotInfo, Views } from 'react-big-calendar'
import { modalBody } from '../ui/__styles__/styles'

interface CalendarProps {
  events: Event[]
  reservedAppointment: (event: SlotInfo) => void
  openEditAppointment: (event: Event) => void
}

const localizer = dateFnsLocalizer({
  format,
  parse,
  startOfWeek,
  getDay,
  locales: { 'en-AU': enAU },
})

const CalendarModalBody: React.FC<CalendarProps> = ({ events, reservedAppointment, openEditAppointment }) => {
  return (
    <div className={modalBody}>
      <Calendar
        selectable
        localizer={localizer}
        defaultDate={new Date(Date.now())}
        defaultView={Views.WEEK}
        views={['month', 'week', 'day']}
        events={events}
        step={30}
        timeslots={12}
        onSelectSlot={reservedAppointment}
        onSelectEvent={openEditAppointment}
        dayLayoutAlgorithm="no-overlap"
      />
    </div>
  )
}

export default CalendarModalBody
