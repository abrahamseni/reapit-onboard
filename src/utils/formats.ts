import { format } from 'date-fns'

export function formatDate(date: string | number | Date, formatType = 'MMMM dd yyyy, HH:mm a') {
  return format(new Date(date), formatType)
}

export function formatCurrency(currency: string, amount: number) {
  return new Intl.NumberFormat(navigator.language, {
    style: 'currency',
    currency: currency,
  }).format(amount)
}
