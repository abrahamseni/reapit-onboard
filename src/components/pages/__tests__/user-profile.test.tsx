import React from 'react'
import { renderWithClient } from '../../../utils/testUtils'
import UserProfile from '../user-profile'

describe('UserProfile', () => {
  it('should match a snapshot', () => {
    expect(renderWithClient(<UserProfile />)).toMatchSnapshot()
  })
})
