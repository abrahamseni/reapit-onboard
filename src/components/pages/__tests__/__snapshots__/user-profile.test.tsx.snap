// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserProfile should match a snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <mock-styled-section>
        <mock-styled-button>
          Go back
        </mock-styled-button>
        <mock-styled-div>
          <mock-styled-div>
            <mock-styled-h1>
              UserProfile
            </mock-styled-h1>
            <mock-styled-h2>
              Profile Information
            </mock-styled-h2>
            <mock-styled-div>
              <mock-styled-div>
                <div>
                  <mock-styled-img
                    alt="user"
                    src="https://images.unsplash.com/photo-1573600073955-f15b3b6caab7?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=415&q=80"
                  />
                </div>
                <mock-styled-p />
                <mock-styled-div>
                  <mock-styled-form>
                    <mock-styled-div>
                      <mock-styled-div>
                        <mock-styled-h2>
                          Personal Details
                        </mock-styled-h2>
                      </mock-styled-div>
                      <mock-styled-div>
                        <mock-styled-div>
                          <mock-styled-input
                            id="test-static-id"
                            type="text"
                          />
                          <mock-styled-span>
                            <svg
                              fill="none"
                              height="1em"
                              viewBox="0 0 20 20"
                              width="1em"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <g>
                                <path
                                  clip-rule="evenodd"
                                  d="M19.3749 10.3316C19.215 10.3316 19.055 10.2665 18.933 10.1363L18.4375 9.60771V18.0831C18.4375 19.14 17.6314 19.9999 16.6406 19.9999H12.5087C12.5058 20 12.5029 20 12.5 20H7.49997C7.49705 20 7.49413 20 7.49121 19.9999H3.35938C2.3686 19.9999 1.5625 19.1401 1.5625 18.0831V9.6077L1.06694 10.1363C0.822878 10.3967 0.427137 10.3967 0.183075 10.1363C-0.0610252 9.87599 -0.0610252 9.45384 0.183075 9.19349L7.95602 0.901689C9.08304 -0.300563 10.9169 -0.300563 12.0439 0.901689L19.8168 9.19345C20.0609 9.45384 20.0609 9.87595 19.8168 10.1363C19.6948 10.2665 19.5349 10.3316 19.3749 10.3316ZM11.875 18.6665H8.12497V13.4162C8.12497 13.0026 8.4404 12.6662 8.8281 12.6662H11.1719C11.5595 12.6662 11.875 13.0026 11.875 13.4162V18.6665ZM13.125 18.6665V13.4162C13.125 12.2674 12.2488 11.3327 11.1719 11.3327H8.8281C7.75115 11.3327 6.87497 12.2674 6.87497 13.4162V18.6665H3.35938C3.05781 18.6665 2.8125 18.4048 2.8125 18.0831V8.27427L8.83988 1.84459C9.47956 1.16221 10.5204 1.16221 11.16 1.84459L17.1875 8.27429V18.0831C17.1875 18.4048 16.9422 18.6665 16.6406 18.6665H13.125Z"
                                  fill="currentColor"
                                  fill-rule="evenodd"
                                />
                              </g>
                              <defs>
                                <clippath
                                  id="clip0"
                                >
                                  <rect
                                    fill="white"
                                    height="1em"
                                    width="1em"
                                  />
                                </clippath>
                              </defs>
                            </svg>
                          </mock-styled-span>
                          <mock-styled-label
                            htmlfor="test-static-id"
                          >
                            Name
                          </mock-styled-label>
                        </mock-styled-div>
                      </mock-styled-div>
                      <mock-styled-div>
                        <mock-styled-div>
                          <mock-styled-input
                            id="test-static-id"
                            type="text"
                          />
                          <mock-styled-span>
                            <svg
                              fill="none"
                              height="1em"
                              viewBox="0 0 20 20"
                              width="1em"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <g>
                                <path
                                  d="M18.125 2.5H1.87501C0.841067 2.5 0 3.34107 0 4.37501V15.625C0 16.659 0.841067 17.5 1.87501 17.5H18.125C19.1589 17.5 20 16.659 20 15.625V4.37501C20 3.34107 19.1589 2.5 18.125 2.5ZM18.125 3.74999C18.2099 3.74999 18.2907 3.76761 18.3645 3.79838L10 11.048L1.63542 3.79838C1.70927 3.76765 1.79005 3.74999 1.87497 3.74999H18.125ZM18.125 16.25H1.87501C1.53017 16.25 1.24999 15.9698 1.24999 15.625V5.11901L9.59045 12.3474C9.70826 12.4493 9.85413 12.5 10 12.5C10.1459 12.5 10.2917 12.4494 10.4096 12.3474L18.75 5.11901V15.625C18.75 15.9698 18.4698 16.25 18.125 16.25Z"
                                  fill="currentColor"
                                />
                              </g>
                              <defs>
                                <clippath
                                  id="clip0"
                                >
                                  <rect
                                    fill="white"
                                    height="1em"
                                    width="1em"
                                  />
                                </clippath>
                              </defs>
                            </svg>
                          </mock-styled-span>
                          <mock-styled-label
                            htmlfor="test-static-id"
                          >
                            Email
                          </mock-styled-label>
                        </mock-styled-div>
                      </mock-styled-div>
                      <mock-styled-div>
                        <mock-styled-div>
                          <mock-styled-input
                            id="test-static-id"
                            type="number"
                          />
                          <mock-styled-span>
                            <svg
                              fill="none"
                              height="1em"
                              viewBox="0 0 20 20"
                              width="1em"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                d="M2.16715 0.897024C2.12211 0.945168 2.04601 1.04573 1.99515 1.11484C1.04974 2.36853 -0.0800943 4.29584 0.00454623 6.15599C0.148202 9.27061 4.3515 13.4102 5.63663 14.6049C6.92216 15.7992 11.3576 19.6891 14.4715 19.6045C14.9906 19.5917 15.5407 19.4826 16.106 19.2799C17.411 18.8121 18.589 17.9222 19.3371 17.2672C19.3371 17.2672 19.4983 17.1301 19.5604 17.0641C19.7169 16.8952 19.8407 16.6487 19.9389 16.3105C19.989 16.1377 19.9999 15.9556 19.9999 15.8337C19.9999 15.6978 19.9863 15.4346 19.897 15.195C19.7747 14.8674 19.6345 14.6329 19.4668 14.4752C19.3426 14.3599 16.5351 12.1604 16.5172 12.148L16.4959 12.134C16.3437 12.0171 16.1849 11.9259 16.0233 11.8626C15.6001 11.6968 15.1315 11.6902 14.7033 11.8432C14.4311 11.9407 14.1861 12.1018 13.997 12.306L13.0322 13.3388C12.9417 13.4358 12.7736 13.5364 12.7142 13.5508C12.6517 13.5624 12.6082 13.5655 12.5803 13.5663L6.10487 7.54868C6.08934 7.53392 6.08158 7.51956 6.07731 7.50946C6.08041 7.50014 6.0843 7.49199 6.08624 7.48578C6.10992 7.43763 6.22174 7.28466 6.36151 7.13324L7.31624 6.0997C7.82137 5.55652 7.94677 4.71167 7.6253 4.05279C7.51076 3.8179 7.3539 3.6424 7.2813 3.5597L7.24364 3.51311C7.225 3.49176 4.86128 0.897799 4.75529 0.800346C4.49243 0.556132 4.07855 0.424124 3.49034 0.39617C3.16342 0.380639 2.75342 0.476927 2.51542 0.624853C2.36244 0.720753 2.24791 0.810054 2.16715 0.897024ZM3.95042 1.64365C4.08282 1.77798 6.23416 4.13433 6.34676 4.25779L6.42596 4.35369C6.45819 4.3898 6.5339 4.47483 6.57816 4.56491C6.69309 4.80097 6.64805 5.10847 6.46168 5.3092L5.50579 6.34391C5.39397 6.46466 5.12219 6.77371 5.01425 7.03113C4.94281 7.20274 4.90904 7.35222 4.90865 7.52577C4.92029 7.85734 5.06084 8.1664 5.30855 8.39974L11.8492 14.4779C12.098 14.709 12.4599 14.7827 12.925 14.6969C13.2846 14.6305 13.674 14.3587 13.8836 14.1343L14.85 13.0996C14.9145 13.0301 14.9999 12.975 15.0962 12.94C15.2569 12.8826 15.4402 12.8853 15.5986 12.9474C15.691 12.9835 15.7679 13.0437 15.7931 13.0635C15.8172 13.0821 15.8366 13.0957 15.8541 13.1066C16.1231 13.3151 18.5373 15.2125 18.6682 15.3228C18.6682 15.3228 18.7256 15.3876 18.806 15.6023C18.8173 15.6326 18.8351 15.7219 18.8351 15.8333C18.8351 15.9137 18.8262 15.9661 18.8204 15.9852C18.7598 16.1933 18.7097 16.2678 18.7097 16.2678C18.672 16.3055 18.5936 16.3703 18.5804 16.3816C17.8975 16.9791 16.8461 17.777 15.7127 18.1831C15.2647 18.3438 14.8376 18.43 14.4416 18.4397C12.3143 18.4976 8.7959 15.9498 6.42985 13.7511C4.06341 11.5528 1.266 8.22968 1.16816 6.10241C1.09672 4.56879 2.09416 2.9183 2.91532 1.82885L2.94677 1.78691C2.96502 1.76206 2.99802 1.71547 3.01511 1.69528C3.04423 1.66772 3.08771 1.64093 3.12227 1.61919C3.17391 1.59278 3.33426 1.55551 3.43443 1.56056C3.77842 1.57725 3.93761 1.63899 3.95042 1.64365Z"
                                fill="currentColor"
                              />
                            </svg>
                          </mock-styled-span>
                          <mock-styled-label
                            htmlfor="test-static-id"
                          >
                            Mobile Phone
                          </mock-styled-label>
                        </mock-styled-div>
                      </mock-styled-div>
                      <mock-styled-div>
                        <mock-styled-div>
                          <mock-styled-input
                            id="test-static-id"
                            type="number"
                          />
                          <mock-styled-span>
                            <svg
                              fill="none"
                              height="1em"
                              viewBox="0 0 20 20"
                              width="1em"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                d="M2.16715 0.897024C2.12211 0.945168 2.04601 1.04573 1.99515 1.11484C1.04974 2.36853 -0.0800943 4.29584 0.00454623 6.15599C0.148202 9.27061 4.3515 13.4102 5.63663 14.6049C6.92216 15.7992 11.3576 19.6891 14.4715 19.6045C14.9906 19.5917 15.5407 19.4826 16.106 19.2799C17.411 18.8121 18.589 17.9222 19.3371 17.2672C19.3371 17.2672 19.4983 17.1301 19.5604 17.0641C19.7169 16.8952 19.8407 16.6487 19.9389 16.3105C19.989 16.1377 19.9999 15.9556 19.9999 15.8337C19.9999 15.6978 19.9863 15.4346 19.897 15.195C19.7747 14.8674 19.6345 14.6329 19.4668 14.4752C19.3426 14.3599 16.5351 12.1604 16.5172 12.148L16.4959 12.134C16.3437 12.0171 16.1849 11.9259 16.0233 11.8626C15.6001 11.6968 15.1315 11.6902 14.7033 11.8432C14.4311 11.9407 14.1861 12.1018 13.997 12.306L13.0322 13.3388C12.9417 13.4358 12.7736 13.5364 12.7142 13.5508C12.6517 13.5624 12.6082 13.5655 12.5803 13.5663L6.10487 7.54868C6.08934 7.53392 6.08158 7.51956 6.07731 7.50946C6.08041 7.50014 6.0843 7.49199 6.08624 7.48578C6.10992 7.43763 6.22174 7.28466 6.36151 7.13324L7.31624 6.0997C7.82137 5.55652 7.94677 4.71167 7.6253 4.05279C7.51076 3.8179 7.3539 3.6424 7.2813 3.5597L7.24364 3.51311C7.225 3.49176 4.86128 0.897799 4.75529 0.800346C4.49243 0.556132 4.07855 0.424124 3.49034 0.39617C3.16342 0.380639 2.75342 0.476927 2.51542 0.624853C2.36244 0.720753 2.24791 0.810054 2.16715 0.897024ZM3.95042 1.64365C4.08282 1.77798 6.23416 4.13433 6.34676 4.25779L6.42596 4.35369C6.45819 4.3898 6.5339 4.47483 6.57816 4.56491C6.69309 4.80097 6.64805 5.10847 6.46168 5.3092L5.50579 6.34391C5.39397 6.46466 5.12219 6.77371 5.01425 7.03113C4.94281 7.20274 4.90904 7.35222 4.90865 7.52577C4.92029 7.85734 5.06084 8.1664 5.30855 8.39974L11.8492 14.4779C12.098 14.709 12.4599 14.7827 12.925 14.6969C13.2846 14.6305 13.674 14.3587 13.8836 14.1343L14.85 13.0996C14.9145 13.0301 14.9999 12.975 15.0962 12.94C15.2569 12.8826 15.4402 12.8853 15.5986 12.9474C15.691 12.9835 15.7679 13.0437 15.7931 13.0635C15.8172 13.0821 15.8366 13.0957 15.8541 13.1066C16.1231 13.3151 18.5373 15.2125 18.6682 15.3228C18.6682 15.3228 18.7256 15.3876 18.806 15.6023C18.8173 15.6326 18.8351 15.7219 18.8351 15.8333C18.8351 15.9137 18.8262 15.9661 18.8204 15.9852C18.7598 16.1933 18.7097 16.2678 18.7097 16.2678C18.672 16.3055 18.5936 16.3703 18.5804 16.3816C17.8975 16.9791 16.8461 17.777 15.7127 18.1831C15.2647 18.3438 14.8376 18.43 14.4416 18.4397C12.3143 18.4976 8.7959 15.9498 6.42985 13.7511C4.06341 11.5528 1.266 8.22968 1.16816 6.10241C1.09672 4.56879 2.09416 2.9183 2.91532 1.82885L2.94677 1.78691C2.96502 1.76206 2.99802 1.71547 3.01511 1.69528C3.04423 1.66772 3.08771 1.64093 3.12227 1.61919C3.17391 1.59278 3.33426 1.55551 3.43443 1.56056C3.77842 1.57725 3.93761 1.63899 3.95042 1.64365Z"
                                fill="currentColor"
                              />
                            </svg>
                          </mock-styled-span>
                          <mock-styled-label
                            htmlfor="test-static-id"
                          >
                            Work Phone
                          </mock-styled-label>
                        </mock-styled-div>
                      </mock-styled-div>
                      <mock-styled-div>
                        <mock-styled-div>
                          <mock-styled-input
                            id="test-static-id"
                            type="checkbox"
                          />
                          <mock-styled-label
                            htmlfor="test-static-id"
                          >
                            Active
                          </mock-styled-label>
                        </mock-styled-div>
                      </mock-styled-div>
                    </mock-styled-div>
                    <mock-styled-button
                      disabled="true"
                    >
                      Save
                    </mock-styled-button>
                  </mock-styled-form>
                </mock-styled-div>
              </mock-styled-div>
            </mock-styled-div>
          </mock-styled-div>
        </mock-styled-div>
      </mock-styled-section>
    </div>
  </body>,
  "container": <div>
    <mock-styled-section>
      <mock-styled-button>
        Go back
      </mock-styled-button>
      <mock-styled-div>
        <mock-styled-div>
          <mock-styled-h1>
            UserProfile
          </mock-styled-h1>
          <mock-styled-h2>
            Profile Information
          </mock-styled-h2>
          <mock-styled-div>
            <mock-styled-div>
              <div>
                <mock-styled-img
                  alt="user"
                  src="https://images.unsplash.com/photo-1573600073955-f15b3b6caab7?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=415&q=80"
                />
              </div>
              <mock-styled-p />
              <mock-styled-div>
                <mock-styled-form>
                  <mock-styled-div>
                    <mock-styled-div>
                      <mock-styled-h2>
                        Personal Details
                      </mock-styled-h2>
                    </mock-styled-div>
                    <mock-styled-div>
                      <mock-styled-div>
                        <mock-styled-input
                          id="test-static-id"
                          type="text"
                        />
                        <mock-styled-span>
                          <svg
                            fill="none"
                            height="1em"
                            viewBox="0 0 20 20"
                            width="1em"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <g>
                              <path
                                clip-rule="evenodd"
                                d="M19.3749 10.3316C19.215 10.3316 19.055 10.2665 18.933 10.1363L18.4375 9.60771V18.0831C18.4375 19.14 17.6314 19.9999 16.6406 19.9999H12.5087C12.5058 20 12.5029 20 12.5 20H7.49997C7.49705 20 7.49413 20 7.49121 19.9999H3.35938C2.3686 19.9999 1.5625 19.1401 1.5625 18.0831V9.6077L1.06694 10.1363C0.822878 10.3967 0.427137 10.3967 0.183075 10.1363C-0.0610252 9.87599 -0.0610252 9.45384 0.183075 9.19349L7.95602 0.901689C9.08304 -0.300563 10.9169 -0.300563 12.0439 0.901689L19.8168 9.19345C20.0609 9.45384 20.0609 9.87595 19.8168 10.1363C19.6948 10.2665 19.5349 10.3316 19.3749 10.3316ZM11.875 18.6665H8.12497V13.4162C8.12497 13.0026 8.4404 12.6662 8.8281 12.6662H11.1719C11.5595 12.6662 11.875 13.0026 11.875 13.4162V18.6665ZM13.125 18.6665V13.4162C13.125 12.2674 12.2488 11.3327 11.1719 11.3327H8.8281C7.75115 11.3327 6.87497 12.2674 6.87497 13.4162V18.6665H3.35938C3.05781 18.6665 2.8125 18.4048 2.8125 18.0831V8.27427L8.83988 1.84459C9.47956 1.16221 10.5204 1.16221 11.16 1.84459L17.1875 8.27429V18.0831C17.1875 18.4048 16.9422 18.6665 16.6406 18.6665H13.125Z"
                                fill="currentColor"
                                fill-rule="evenodd"
                              />
                            </g>
                            <defs>
                              <clippath
                                id="clip0"
                              >
                                <rect
                                  fill="white"
                                  height="1em"
                                  width="1em"
                                />
                              </clippath>
                            </defs>
                          </svg>
                        </mock-styled-span>
                        <mock-styled-label
                          htmlfor="test-static-id"
                        >
                          Name
                        </mock-styled-label>
                      </mock-styled-div>
                    </mock-styled-div>
                    <mock-styled-div>
                      <mock-styled-div>
                        <mock-styled-input
                          id="test-static-id"
                          type="text"
                        />
                        <mock-styled-span>
                          <svg
                            fill="none"
                            height="1em"
                            viewBox="0 0 20 20"
                            width="1em"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <g>
                              <path
                                d="M18.125 2.5H1.87501C0.841067 2.5 0 3.34107 0 4.37501V15.625C0 16.659 0.841067 17.5 1.87501 17.5H18.125C19.1589 17.5 20 16.659 20 15.625V4.37501C20 3.34107 19.1589 2.5 18.125 2.5ZM18.125 3.74999C18.2099 3.74999 18.2907 3.76761 18.3645 3.79838L10 11.048L1.63542 3.79838C1.70927 3.76765 1.79005 3.74999 1.87497 3.74999H18.125ZM18.125 16.25H1.87501C1.53017 16.25 1.24999 15.9698 1.24999 15.625V5.11901L9.59045 12.3474C9.70826 12.4493 9.85413 12.5 10 12.5C10.1459 12.5 10.2917 12.4494 10.4096 12.3474L18.75 5.11901V15.625C18.75 15.9698 18.4698 16.25 18.125 16.25Z"
                                fill="currentColor"
                              />
                            </g>
                            <defs>
                              <clippath
                                id="clip0"
                              >
                                <rect
                                  fill="white"
                                  height="1em"
                                  width="1em"
                                />
                              </clippath>
                            </defs>
                          </svg>
                        </mock-styled-span>
                        <mock-styled-label
                          htmlfor="test-static-id"
                        >
                          Email
                        </mock-styled-label>
                      </mock-styled-div>
                    </mock-styled-div>
                    <mock-styled-div>
                      <mock-styled-div>
                        <mock-styled-input
                          id="test-static-id"
                          type="number"
                        />
                        <mock-styled-span>
                          <svg
                            fill="none"
                            height="1em"
                            viewBox="0 0 20 20"
                            width="1em"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M2.16715 0.897024C2.12211 0.945168 2.04601 1.04573 1.99515 1.11484C1.04974 2.36853 -0.0800943 4.29584 0.00454623 6.15599C0.148202 9.27061 4.3515 13.4102 5.63663 14.6049C6.92216 15.7992 11.3576 19.6891 14.4715 19.6045C14.9906 19.5917 15.5407 19.4826 16.106 19.2799C17.411 18.8121 18.589 17.9222 19.3371 17.2672C19.3371 17.2672 19.4983 17.1301 19.5604 17.0641C19.7169 16.8952 19.8407 16.6487 19.9389 16.3105C19.989 16.1377 19.9999 15.9556 19.9999 15.8337C19.9999 15.6978 19.9863 15.4346 19.897 15.195C19.7747 14.8674 19.6345 14.6329 19.4668 14.4752C19.3426 14.3599 16.5351 12.1604 16.5172 12.148L16.4959 12.134C16.3437 12.0171 16.1849 11.9259 16.0233 11.8626C15.6001 11.6968 15.1315 11.6902 14.7033 11.8432C14.4311 11.9407 14.1861 12.1018 13.997 12.306L13.0322 13.3388C12.9417 13.4358 12.7736 13.5364 12.7142 13.5508C12.6517 13.5624 12.6082 13.5655 12.5803 13.5663L6.10487 7.54868C6.08934 7.53392 6.08158 7.51956 6.07731 7.50946C6.08041 7.50014 6.0843 7.49199 6.08624 7.48578C6.10992 7.43763 6.22174 7.28466 6.36151 7.13324L7.31624 6.0997C7.82137 5.55652 7.94677 4.71167 7.6253 4.05279C7.51076 3.8179 7.3539 3.6424 7.2813 3.5597L7.24364 3.51311C7.225 3.49176 4.86128 0.897799 4.75529 0.800346C4.49243 0.556132 4.07855 0.424124 3.49034 0.39617C3.16342 0.380639 2.75342 0.476927 2.51542 0.624853C2.36244 0.720753 2.24791 0.810054 2.16715 0.897024ZM3.95042 1.64365C4.08282 1.77798 6.23416 4.13433 6.34676 4.25779L6.42596 4.35369C6.45819 4.3898 6.5339 4.47483 6.57816 4.56491C6.69309 4.80097 6.64805 5.10847 6.46168 5.3092L5.50579 6.34391C5.39397 6.46466 5.12219 6.77371 5.01425 7.03113C4.94281 7.20274 4.90904 7.35222 4.90865 7.52577C4.92029 7.85734 5.06084 8.1664 5.30855 8.39974L11.8492 14.4779C12.098 14.709 12.4599 14.7827 12.925 14.6969C13.2846 14.6305 13.674 14.3587 13.8836 14.1343L14.85 13.0996C14.9145 13.0301 14.9999 12.975 15.0962 12.94C15.2569 12.8826 15.4402 12.8853 15.5986 12.9474C15.691 12.9835 15.7679 13.0437 15.7931 13.0635C15.8172 13.0821 15.8366 13.0957 15.8541 13.1066C16.1231 13.3151 18.5373 15.2125 18.6682 15.3228C18.6682 15.3228 18.7256 15.3876 18.806 15.6023C18.8173 15.6326 18.8351 15.7219 18.8351 15.8333C18.8351 15.9137 18.8262 15.9661 18.8204 15.9852C18.7598 16.1933 18.7097 16.2678 18.7097 16.2678C18.672 16.3055 18.5936 16.3703 18.5804 16.3816C17.8975 16.9791 16.8461 17.777 15.7127 18.1831C15.2647 18.3438 14.8376 18.43 14.4416 18.4397C12.3143 18.4976 8.7959 15.9498 6.42985 13.7511C4.06341 11.5528 1.266 8.22968 1.16816 6.10241C1.09672 4.56879 2.09416 2.9183 2.91532 1.82885L2.94677 1.78691C2.96502 1.76206 2.99802 1.71547 3.01511 1.69528C3.04423 1.66772 3.08771 1.64093 3.12227 1.61919C3.17391 1.59278 3.33426 1.55551 3.43443 1.56056C3.77842 1.57725 3.93761 1.63899 3.95042 1.64365Z"
                              fill="currentColor"
                            />
                          </svg>
                        </mock-styled-span>
                        <mock-styled-label
                          htmlfor="test-static-id"
                        >
                          Mobile Phone
                        </mock-styled-label>
                      </mock-styled-div>
                    </mock-styled-div>
                    <mock-styled-div>
                      <mock-styled-div>
                        <mock-styled-input
                          id="test-static-id"
                          type="number"
                        />
                        <mock-styled-span>
                          <svg
                            fill="none"
                            height="1em"
                            viewBox="0 0 20 20"
                            width="1em"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M2.16715 0.897024C2.12211 0.945168 2.04601 1.04573 1.99515 1.11484C1.04974 2.36853 -0.0800943 4.29584 0.00454623 6.15599C0.148202 9.27061 4.3515 13.4102 5.63663 14.6049C6.92216 15.7992 11.3576 19.6891 14.4715 19.6045C14.9906 19.5917 15.5407 19.4826 16.106 19.2799C17.411 18.8121 18.589 17.9222 19.3371 17.2672C19.3371 17.2672 19.4983 17.1301 19.5604 17.0641C19.7169 16.8952 19.8407 16.6487 19.9389 16.3105C19.989 16.1377 19.9999 15.9556 19.9999 15.8337C19.9999 15.6978 19.9863 15.4346 19.897 15.195C19.7747 14.8674 19.6345 14.6329 19.4668 14.4752C19.3426 14.3599 16.5351 12.1604 16.5172 12.148L16.4959 12.134C16.3437 12.0171 16.1849 11.9259 16.0233 11.8626C15.6001 11.6968 15.1315 11.6902 14.7033 11.8432C14.4311 11.9407 14.1861 12.1018 13.997 12.306L13.0322 13.3388C12.9417 13.4358 12.7736 13.5364 12.7142 13.5508C12.6517 13.5624 12.6082 13.5655 12.5803 13.5663L6.10487 7.54868C6.08934 7.53392 6.08158 7.51956 6.07731 7.50946C6.08041 7.50014 6.0843 7.49199 6.08624 7.48578C6.10992 7.43763 6.22174 7.28466 6.36151 7.13324L7.31624 6.0997C7.82137 5.55652 7.94677 4.71167 7.6253 4.05279C7.51076 3.8179 7.3539 3.6424 7.2813 3.5597L7.24364 3.51311C7.225 3.49176 4.86128 0.897799 4.75529 0.800346C4.49243 0.556132 4.07855 0.424124 3.49034 0.39617C3.16342 0.380639 2.75342 0.476927 2.51542 0.624853C2.36244 0.720753 2.24791 0.810054 2.16715 0.897024ZM3.95042 1.64365C4.08282 1.77798 6.23416 4.13433 6.34676 4.25779L6.42596 4.35369C6.45819 4.3898 6.5339 4.47483 6.57816 4.56491C6.69309 4.80097 6.64805 5.10847 6.46168 5.3092L5.50579 6.34391C5.39397 6.46466 5.12219 6.77371 5.01425 7.03113C4.94281 7.20274 4.90904 7.35222 4.90865 7.52577C4.92029 7.85734 5.06084 8.1664 5.30855 8.39974L11.8492 14.4779C12.098 14.709 12.4599 14.7827 12.925 14.6969C13.2846 14.6305 13.674 14.3587 13.8836 14.1343L14.85 13.0996C14.9145 13.0301 14.9999 12.975 15.0962 12.94C15.2569 12.8826 15.4402 12.8853 15.5986 12.9474C15.691 12.9835 15.7679 13.0437 15.7931 13.0635C15.8172 13.0821 15.8366 13.0957 15.8541 13.1066C16.1231 13.3151 18.5373 15.2125 18.6682 15.3228C18.6682 15.3228 18.7256 15.3876 18.806 15.6023C18.8173 15.6326 18.8351 15.7219 18.8351 15.8333C18.8351 15.9137 18.8262 15.9661 18.8204 15.9852C18.7598 16.1933 18.7097 16.2678 18.7097 16.2678C18.672 16.3055 18.5936 16.3703 18.5804 16.3816C17.8975 16.9791 16.8461 17.777 15.7127 18.1831C15.2647 18.3438 14.8376 18.43 14.4416 18.4397C12.3143 18.4976 8.7959 15.9498 6.42985 13.7511C4.06341 11.5528 1.266 8.22968 1.16816 6.10241C1.09672 4.56879 2.09416 2.9183 2.91532 1.82885L2.94677 1.78691C2.96502 1.76206 2.99802 1.71547 3.01511 1.69528C3.04423 1.66772 3.08771 1.64093 3.12227 1.61919C3.17391 1.59278 3.33426 1.55551 3.43443 1.56056C3.77842 1.57725 3.93761 1.63899 3.95042 1.64365Z"
                              fill="currentColor"
                            />
                          </svg>
                        </mock-styled-span>
                        <mock-styled-label
                          htmlfor="test-static-id"
                        >
                          Work Phone
                        </mock-styled-label>
                      </mock-styled-div>
                    </mock-styled-div>
                    <mock-styled-div>
                      <mock-styled-div>
                        <mock-styled-input
                          id="test-static-id"
                          type="checkbox"
                        />
                        <mock-styled-label
                          htmlfor="test-static-id"
                        >
                          Active
                        </mock-styled-label>
                      </mock-styled-div>
                    </mock-styled-div>
                  </mock-styled-div>
                  <mock-styled-button
                    disabled="true"
                  >
                    Save
                  </mock-styled-button>
                </mock-styled-form>
              </mock-styled-div>
            </mock-styled-div>
          </mock-styled-div>
        </mock-styled-div>
      </mock-styled-div>
    </mock-styled-section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
